androguard/cli.py

```diff
@@ -0,0 +1,26 @@
+# 3rd party modules
+from lxml import etree
+
+# internal modules
+from androguard.core import androconf
+from androguard.core.bytecodes import apk
+from androguard.util import read
+
+
+def androaxml_main(inp, outp=None):
+    ret_type = androconf.is_android(inp)
+    if ret_type == "APK":
+        a = apk.APK(inp)
+        axml = a.get_android_manifest_xml()
+    elif ".xml" in inp:
+        axml = apk.AXMLPrinter(read(inp)).get_xml_obj()
+    else:
+        print("Unknown file type")
+        return
+
+    buff = etree.tostring(axml, pretty_print=True, encoding="utf-8")
+    if outp:
+        with open(outp, "wb") as fd:
+            fd.write(buff)
+    else:
+        print(buff.decode("UTF-8"))
```

Comment:
click.echo should deal with bytes/str as well, right?

Reply:
Hmm... for this PR, I would like not to change this. I have never used `click.echo` and I never had problems with `print`. So if it worked before, it will still work. If it was broken before, it will still be broken.

Reply:
okay, no worries! Leave it as it is then. I was just reading through the click doc and they are advertising this function as the mighty saviour :smile: